.github/ISSUE_TEMPLATE/ui-copy-request.md

```diff
@@ -0,0 +1,38 @@
+---
+name: UI Copy Request
+about: Describe the UI text that you need us to work on.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Description 
+[Summarize your request here. What is the work, what is its current status, and how can we help? Feel free to include screenshots.]
+
+### Software version
+Does this work affect serverless, stateful/classic, or both? Is this work targeted for a particular release version? Is this work related to text that's already in production?
+
+### Timeline
+What is the timeline for this work? When would it be ideal for for us to complete the request? What deliverables do you need, and when? If applicable, how do the release timelines for this feature differ between serverless / stateful?
+
+### Collaborators
+Please include contact information for the responsible product manager, designer, and developer (name, github / slack @):
+
+PM:
+Designer: 
+Developer:
+
+## Links / related assets  
+
+Please include each of the following, if applicable:
+
+Figma link(s):
+
+Github epic link(s):
+
+Github issue link(s):
+
+Production page(s) where content appears (link, or describe how to find it):
```

Comment:
What a production page? Are we just asking for a way to test flows and see copy? Also, we probably want to encourage some security best practices here, so we're not sharing internal deployments. Maybe a line or two that encourages the secure sharing of test environments?